500.html

```diff
@@ -1,219 +1,376 @@
-									<li class="hidden-sm hidden-xs"><a href="https://blog.fossasia.org/contact/" target="_self">Contact</a></li>											
-									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
-									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
-      								<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
 
+<head>
+	<meta charset="utf-8">
+	<title>FOSSASIA | Asia's Open Technology Organization for Open Source Software, Hardware, Open Design, Open Data, Open Knowledge</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta name="description" content="FOSSASIA Open Tech Community - Open Source, Big Data, Design Thinking and Free Knowledge Tools with Asia's Premier Open Technology Organization">
+	<meta name="keywords" content="FOSSASIA, FOSS, Free and Open Source Software, Asia">
+	<meta name="author" content="FOSSASIA">
+	<meta property="og:title" content="FOSSASIA" />
+	<meta property="og:type" content="website" />
+	<meta property="og:image" content="http://fossasia.github.io/fossasia15/img/fossasia-dark.png" />
+	<meta name="twitter:card" content="summary">
+	<meta name="twitter:site" content="@fossasia">
+	<meta name="twitter:title" content="FOSSASIA">
+	<meta name="twitter:description" content="FOSSASIA Open Technology Community - Open Source, Big Data, Design Thinking and Free Knowledge Tools with Asia's Premier Open Technology Organization">
+	<meta name="twitter:creator" content="@fossasia">
+	<meta name="twitter:image" content="http://fossasia.github.io/fossasia15/img/fossasia-dark.png">
+
+	<link rel="shortcut icon" href="fossasia.ico" type="image/x-icon" />
+	<link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/flexslider.min.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/elegant-icons.min.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/pe-icon-7-stroke.min.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/custom-blog.css" rel="stylesheet" type="text/css" media="all" />
+	<link href="css/bootstrap-theme.min.css" rel="stylesheet" type="text/css" media="all" />
+
+	<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+
+	<script src="js/google-analytics-500.js"></script>
```

Comment:
>I would be glad if you could limit these changes to a single file as well, that would be enough for the task instead of intending a large part of the repo.

" Simple typos or one-line code changes are not sufficient. "